stdlib/LinearAlgebra/src/lapack.jl

```diff
@@ -552,7 +552,7 @@ for (gebrd, gelqf, geqlf, geqrf, geqp3, geqrt, geqrt3, gerqf, getrf, elty, relty
                       m, n, A, max(1,stride(A,2)), tau, work, lwork, info)
                 chklapackerror(info[])
                 if i == 1
-                    lwork = BlasInt(real(work[1]))
+                    lwork = max(m, BlasInt(1), BlasInt(real(work[1])))
```

Comment:
If we use 32bit version BLAS on 64bit system,
 `max(m, BlasInt(1), BlasInt(real(work[1])))` will return a `Int64`.
Replace this with `max(BlasInt(m), BlasInt(1), BlasInt(real(work[1])))`?

Reply:
I  am using `max(BlasInt(m),BlasInt(real(work[1]))) ` since the returned _optimal_ value is always at least 1.